src/routes/[dictionaryId]/_SideMenu.svelte

```diff
@@ -70,6 +74,7 @@
         {$_('misc.settings', { default: 'Settings' })}
       </span>
     </a>
+    <ExportingButton images={imgs} />
```

Comment:
Let's put this behind an admin flag instead of manager for the moment so we can merge it and try it for a bit before releasing. We often stagger the rollout in that manner.